org-code-javabuilder/protocol/src/testFixtures/java/org/code/protocol/GlobalProtocolTestFactory.java

```diff
@@ -61,6 +65,16 @@ public Builder withLifecycleNotifier(LifecycleNotifier lifecycleNotifier) {
       return this;
     }
 
+    public Builder withContentManager(ContentManager contentManager) {
```

Comment:
is this method used anywhere?

Reply:
Not currently - just added it in case unit tests would need to mock this for any reason.